libs/testnorge-common-command/src/main/java/no/nav/registre/testnorge/libs/common/command/GetOrganisasjonCommand.java

```diff
@@ -37,7 +38,7 @@ public OrganisasjonDTO call() {
                     .retrieve()
                     .bodyToMono(OrganisasjonDTO.class)
                     .block();
-        } catch (HttpClientErrorException.NotFound e) {
+        } catch (HttpClientErrorException.NotFound | WebClientResponseException.NotFound e) {
```

Comment:
Du kan vel fjerne `HttpClientErrorException.NotFound` siden jeg tror det er for restTemplate

Reply:
Aha, er det som er forskjellen, ja 💡